### DISCUSSION TITLE
Build fails with Sentry when imported filename is loading

### DISCUSSION THREAD

**Discussion by cnotethegr8** [vercel/next.js#48468] [Help]
### Summary

I experienced a bug with `@sentry/nextjs` where `next build` will fail if a page has an imported component whose filename is `loading.tsx`. I opened a ticket with Sentry [here](https://github.com/getsentry/sentry-javascript/issues/7869).

Upon investigation, the maintainer believes the issue is with the Next.js interop settings. That comment can be found [here](https://github.com/getsentry/sentry-javascript/issues/7869#issuecomment-1510270736).

With that information I tested if the problem would persist when the filename is any of the other Next.js app router reserved names (ie. `layout.tsx`, `page.tsx`, etc). Sure enough, the problem exists when using these filenames.

It should also be noted, while I'm using `next@13.3.0`, I'm still using the old `pages` router.

**Comment by frangiskos** [comment]
I have exactly the same issue here. Using `next@13.5.1` with the `pages` router
